ui/src/lib/gql/user/useQueries.ts

```diff
@@ -26,7 +26,23 @@ export const useQueries = () => {
     staleTime: Infinity,
   });
 
+  // Not using react-query because no observers are needed on this
+  const searchUserQuery = async (email: string) => {
```

Comment:
It doesn't have to be email, right? It can be username too I thought. If I am right, make email -> queryString or searchTerm

Reply:
Me and Piyush discussed and we decided it's best to use email since multiple users can exist with the the same username. 
The PR to be merged is open below. 
https://github.com/reearth/reearthx/pull/41

Reply:
Oh okay. Awesome